src/lib.rs

```diff
@@ -38,3 +38,7 @@ mod syscalls;
 #[cfg(any(target_arch = "arm", target_arch = "riscv32"))]
 #[global_allocator]
 static ALLOCATOR: linked_list_allocator::LockedHeap = linked_list_allocator::LockedHeap::empty();
+
+// Dummy structure to force importing the panic_handler and other no_std elements when nothing else
+// is imported.
+pub struct LibTock {}
```

Comment:
Does this work as a unit-like struct?

Reply:
This is a zero-sized-type (https://doc.rust-lang.org/nomicon/exotic-sizes.html#zero-sized-types-zsts), which should have no effect on the generated code (apart from allowing the import). I had to introduce it because otherwise Rust would complain about an unused import when adding `use libtock;` in the example app.

I'm not sure if there's a better or known pattern for this though (e.g. `pub const IMPORT_ME = ()`), as I haven't encountered this problem in Rust before.

Reply:
By unit-like struct, I'm referring to `pub struct LibTock;`, which removes the braces from the syntax.

Reply:
Oh, yes I missed that that syntax existed. Fixed.